router/batchrouter/asyncdestinationmanager/asyncdestinationmanager.go

```diff
@@ -18,7 +19,7 @@ import (
 )
 
 type Asyncdestinationmanager interface {
-	Upload(url string, filePath string, config map[string]interface{}, destType string, failedJobIDs []int64, importingJobIDs []int64, destinationID string) common.AsyncUploadOutput
+	Upload(destination *backendconfig.DestinationT, asyncDestStruct map[string]*common.AsyncDestinationStruct) common.AsyncUploadOutput
```

Comment:
Upload(destination *backendconfig.DestinationT, *common.AsyncDestinationStruct) is better

Reply:
updated